tests/integration_frameworks/llm/openai/test_openai_llmobs.py

```diff
@@ -478,14 +480,14 @@ def test_responses_create_reasoning(
                 tool_choice="auto",
                 truncation="disabled",
                 text={"format": {"type": "text"}, "verbosity": "medium"},
-                reasoning_tokens=mock.ANY,
                 stream=stream,
             ),
             metrics={
                 "input_tokens": mock.ANY,
                 "output_tokens": mock.ANY,
                 "total_tokens": mock.ANY,
                 "cache_read_input_tokens": mock.ANY,
+                "reasoning_output_tokens": mock.ANY,
```

Comment:
maybe because this one is the one where we actually expect a non-zero number of reasoning tokens, we can use the real count? i think it was 64, but if it fails locally you can just update it with whatever it said the output was (should we ever regress) (it also mighhttt be different for `stream=True` vs `stream=False`)